backend/kernelCI_app/views/treeTestsView.py

```diff
@@ -38,15 +39,59 @@ def extract_error_message(self, misc: Union[str, dict, None]):
         print("unknown error_msg in misc", misc)
         return "unknown error"
 
-    def get(self, request, commit_hash: str | None):
-        path_param = request.GET.get('path')
+    def __mapStatusInClause(self, value):
+        if value is not list:
+            value = [value,]
+        status = []
+        for i in value:
+            if i == "Success":
```

Comment:
test status should not be grouped, we must have a filter for each status (MISS, DONE, SKIP....)